RevenueCatUI/Templates/V2/Components/ComponentsView.swift

```diff
@@ -62,8 +62,6 @@ struct ComponentsView: View {
                     PackageComponentView(viewModel: viewModel, onDismiss: onDismiss)
                 case .purchaseButton(let viewModel):
                     PurchaseButtonComponentView(viewModel: viewModel)
-                case .stickyFooter(let viewModel):
-                    StickyFooterComponentView(viewModel: viewModel)
```

Comment:
Sticky footer is not a standalone component (not in the component tree) so removed it from here. I did this in this PR since the navigation bar fell in the same category 🤷‍♂️